.github/workflows/longtest.yml

```
@@ -11,16 +11,20 @@ on:
           - all
           - gadi
           - setonix
+  schedule:
+    - cron: "5 19 * * 0"
```

Did we not say every second Sunday? Or did decide to go every Sunday?
```suggestion
	- cron: "5 19 1-7,15-21  * 0"
```

---

Unfortunately I don't think we can do every other Sunday very easily... We could run a timer on a local machine to trigger a workflow dispatch. As it is, I think this syntax runs it on days 1-7, 15-21 *and* on Sundays. Shame cron syntax is so limited.